zetaclient/chains/base/observer_test.go

```diff
@@ -175,15 +175,6 @@ func TestObserverGetterAndSetter(t *testing.T) {
 		require.Equal(t, newChain, ob.Chain())
 	})
 
-	t.Run("should be able to update chain params", func(t *testing.T) {
```

Comment:
have we tested `SetChainParams` after renaming?

Reply:
I don't think it requires testing
```go
// SetChainParams attaches a new chain params to the observer.
func (ob *Observer) SetChainParams(params observertypes.ChainParams) {
	ob.mu.Lock()
	defer ob.mu.Unlock()

	ob.chainParams = params
}
```